Source/Dafny/Dafny.atg

```diff
@@ -773,6 +765,7 @@ TOKENS
   ensures = "ensures".
   ghost = "ghost".
   witness = "witness".
+  lbracecolon = "{:".
```

Comment:
Note that the previous grammar treated "{" and ":" as two separate colons (but I don't mind changing this into one token if it helps).